### ISSUE TITLE
css broken

### ISSUE THREAD

**Issue by alekserok** [appzcoder/laravel-admin#57]
laravel 5.6 styles seems to be broken. when i use styles from previously generated for laravel 5.5 project it works fine. may be it is generator itself issue and appears after last commit or may be it is connected with laravel version

**Comment by sohelamin** [comment]
@alekserok Did you try with fresh installation?
I think you haven't newest vendor files. Make sure you have a backend layout file similar to https://github.com/appzcoder/laravel-admin/blob/master/publish/resources/views/layouts/backend.blade.php

**Comment by alekserok** [comment]
yes, i try with fresh laravel 5.6. backend layout file is similar to provided

**Comment by kevin-openc** [comment]
i have tried in a brand new 5.4 laravel installation, the css is also broken.

no error reported in console

<img width="1204" alt="screen shot 2018-04-09 at 10 48 23 am" src="https://user-images.githubusercontent.com/22143564/38477099-90f7fe92-3be3-11e8-9f09-dc02fe51729e.png">

**Comment by kinyou** [comment]
@alekserok @kevin-openc 
First please make sure exec the command   php artisan cache:clear in your app root directory
Second Clean the browser's cache

now it work !
<img width="1279" alt="wx20180414-112215 2x" src="https://user-images.githubusercontent.com/7870804/38763835-a57ba43a-3fd6-11e8-8893-586c92675639.png">

**Comment by alekserok** [comment]
Thank you @kinyou, I tried, but still no result... The issue is in bootstrap version. Laravel default app.css file uses Bootstrap v3.3.7. And [this](https://github.com/appzcoder/laravel-admin/commit/30ed150491a4db4c295e9affe1d7279b7e57b2dd) commit changes all css classes to be used with bootstrap 4

**Comment by kevin-openc** [comment]
it works after upgrade to bootstrap 4

i follow this guideline
https://laracasts.com/discuss/channels/tips/tip-using-bootstrap-4-alpha-6-with-laravel-54/

**Comment by ta9to** [comment]
it works for me
```
$ php artisan view:clear
```
